apis/Google.Cloud.Diagnostics.AspNetCore/docs/index.md

```diff
@@ -64,6 +68,29 @@ property on the `Startup` class to obtain values for `ProjectID`, `Service` or `
 
 {{sample:Logging.UseGoogleLogger}}
 
+## Troubleshooting Logging
+
+Sometimes it is neccessary to diagnose log operations. It might be that logging is failing or that
+we simply cannot find where the logs are being stored in GCP. What follows are a couple of code samples
+that can be useful to find out what might be wrong with logging operations.
+
+### Propagating Exceptions
```

Comment:
I'm not sure whether having a whole extra section here is worth it - maybe just promote the two samples to `###` level, and include the paragraph below directly in this section? (i.e. just remove this header and one `#` from each of the headers below).

Reply:
Sure, no problem.